data/docs/logs-management/send-logs/opentelemetry-nodejs-bunyan-logs.mdx

```diff
@@ -0,0 +1,286 @@
+---
+date: 2025-11-24
+tags: [SigNoz Cloud, Self-Host]
+title: Send logs from Node.js Bunyan logger to SigNoz using OpenTelemetry
+id: opentelemetry-nodejs-bunyan-logs
+doc_type: howto
+description: Learn how to send logs from Node.js Bunyan logger to SigNoz using the OpenTelemetry SDK for OTLP log export.
+---
+
+import GetHelp from '@/components/shared/get-help.md'
+
+## Overview
+
+This guide will walk you through the process of sending logs from a Node.js application that uses the Bunyan logger to SigNoz using the OpenTelemetry SDK for OTLP log export.
+
+## Prerequisites
+
+- Node.js version 14 or higher
+- A SigNoz instance (either SigNoz Cloud or self-hosted)
+- A Node.js application that uses the Bunyan logger
+
+## Setup
+
+### Step 1: Install dependencies
+
+Install the required OpenTelemetry dependencies:
+
+<Tabs>
+<TabItem value="npm" label="npm" default>
+
+```bash
+npm install @opentelemetry/sdk-node \
+  @opentelemetry/exporter-logs-otlp-http \
+  @opentelemetry/instrumentation-bunyan \
+  bunyan \
+  express
+```
+
+</TabItem>
+<TabItem value="yarn" label="yarn">
+
+```bash
+yarn add @opentelemetry/sdk-node \
+  @opentelemetry/exporter-logs-otlp-http \
+  @opentelemetry/instrumentation-bunyan \
+  bunyan \
+  express
+```
+
+</TabItem>
+</Tabs>
+
+### Step 2: Configure OpenTelemetry SDK
+
+Create a file named `instrumentation.js` or `logging.js` in your project root:
+
+```javascript:instrumentation.js
```

Comment:
Name it `logging-instrumentation.js` as instrumentation.js will confuse users who already have instrumentation.js with tracing instrumentation